src/applications/personalization/dashboard/tests/e2e/loa1.cypress.spec.js

```diff
@@ -20,8 +20,12 @@ const loa1DashboardTest = (mobile, stubs) => {
   }
 
   // make sure that the "Verify" alert is shown
-  cy.findByText(/Verify your identity to access/i).should('exist');
-  cy.findByText(/we need to make sure you’re you/i).should('exist');
+  cy.findByText(/We need you to verify your identity for this account/i).should(
```

Comment:
Wondering if these tests are brittle as the text is in an external component. Is there a way to test for the presence without having to specify the text?